sources/name.json

```diff
@@ -3,8 +3,10 @@
 	"fullname": "name",
 	"name": "name",
 	"prefix": "name",
-    	"key":"id",
+	"key": "id",
 	"url": "",
-	"license": "CC0",
-	"description": ""
-}
+	"license": "",
+	"license_type": "",
+	"license_text": "",
+	"description": "A name for a given record."
```

Comment:
@stepps00 Please specify this (and `geom`) is a Who's On First property prefix (not a "source"), and consider not printing them out in the README.md.